tools/paconn-cli/paconn/operations/upsert.py

```diff
@@ -91,6 +94,17 @@ def upsert(powerapps_rp, settings, client_secret, is_update, overwrite_settings)
         elif oauth_settings and not client_secret and not is_update:
             raise CLIError('Please provide OAuth2 client secret using the --secret argument.')
 
+    # Add secret in connection parameter set
+    multi_auth = properties.get(_CONNECTION_PARAMETER_SET, {}).get(_VALUES, [])
+    for auth in multi_auth:
```

Comment:
This will work if single secret is used for all oauth settings in a multi auth case. In future we need to allow separate secrets.